Softeq.XToolkit.Bindings.iOS/ObservableCollectionViewSource.cs

```diff
@@ -43,36 +43,37 @@ public class ObservableCollectionViewSource<TItem, TCell> : UICollectionViewSour
         private UICollectionView _view;
 
         /// <summary>
-        ///     Creates and initializes a new instance of <see cref="ObservableCollectionViewSource{TItem, TCell}" />
+        ///     Initializes a new instance of the <see cref="ObservableCollectionViewSource{TItem, TCell}"/> class.
         /// </summary>
         public ObservableCollectionViewSource()
         {
         }
 
         /// <summary>
-        ///     Creates and initializes a new instance of <see cref="ObservableCollectionViewSource{TItem, TCell}" />
-        ///     with a value for IsInfiniteScroll flag.
+        ///     Initializes a new instance of the <see cref="ObservableCollectionViewSource{TItem, TCell}"/> class
+        ///     with a value for <see cref="IsInfiniteScroll"/> flag.
         /// </summary>
-        public ObservableCollectionViewSource(bool canBeScrolledInfinitely) : this()
+        public ObservableCollectionViewSource(bool canBeScrolledInfinitely)
+            : this()
         {
             IsInfiniteScroll = canBeScrolledInfinitely;
         }
 
         /// <summary>
-        ///     A delegate to a method taking a <see cref="UICollectionViewCell" />
+        ///     Gets or sets a delegate to a method taking a <see cref="T:UIKit.UICollectionViewCell" />
         ///     and setting its elements' properties according to the item
         ///     passed as second parameter.
         /// </summary>
         public Action<TCell, TItem, NSIndexPath> BindCellDelegate { get; set; }
 
         /// <summary>
-        ///     Indicates whether this <see cref="UICollectionViewCell" />
+        ///     Gets a value indicating whether indicates whether this <see cref="T:UIKit.UICollectionViewCell" />
```

Comment:
Looks like a misprint. Probably, it should be something like that: "Gets a value indicating whether this ..."